source/sync/data-model/update-data-model/update-data-model-production.txt

```diff
@@ -0,0 +1,431 @@
+.. _update-data-model-production:
+
+=================================
+Update a Data Model in Production
+=================================
+
+.. meta::
+   :description: Update your data model for an app in production.
+   :keywords: Realm, object model, C++ SDK, Flutter SDK, Java SDK, Kotlin SDK, .NET SDK, Node.js SDK, React Native SDK, Swift SDK
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+After your app goes to production, you might need to make additional updates to
+you data model. Production-level apps can take advantage of schema versioning,
+which supports backward-compatibility with your existing schemas
+when you make certain changes. However, updates should always be made with
+caution, as some data model changes can cause data loss and break connectivity
+with existing client devices.
+
+This feature is enabled by default for new apps created after [DATE TBD].
+
+.. note:: Enabling Schema Versioning in Older Apps
+
+   If your app backend was created *before* [DATE TBD], you must terminate,
+   then re-enable Sync to enable the schema versioning feature. To learn how,
+   refer to :ref:`pause-or-terminate-sync`.
+
+If your app is still in development, we recommend using Development Mode
+to update your data model. To learn how, refer to
```

Comment:
Is it worth adding that this is because there is a cost to schema versioning, or not important / it feels implicit that there would be (its not big, but non-zero)

Reply:
I think so, but might be better to frame it as Dev Mode reducing friction (but only apps in development can take advantage) vs. belaboring the cost of schema versioning (the rest of the page hits that well enough)